server/routes/index.ts

```diff
@@ -18,17 +18,18 @@ export default function routes(services: Services): Router {
   prisonerSummarySetup(router, services)
 
   // Application routes
-  prisonerList(router, services)
+  inPrisonCoursesAndQualifications(router, services)
```

Comment:
I had to re-order the route definition because express has some rules about the sequencing/ordering of how it resolves them and their middlewares etc.

I've not explained that very well, but previously we had `prisonerList` first because it made sense for it to be defined first as it is the first page the user sees and is the root route. The problem though is that it applies the `checkUserHasViewAuthority` middleware to the root:
```
export default (router: Router, services: Services) => {
  const prisonerListController = new PrisonerListController(services.prisonerListService)

  router.use('/', [checkUserHasViewAuthority()])
  router.get('/', [asyncMiddleware(prisonerListController.getPrisonerListView)])
}
```
This means that the user has to be a PLP user (either with our editor or viewer role) in order to see the root route .... or any route below it. That was a problem ...

The `inPrisonCoursesAndQualifications` routes adds a new route `/prisoner/:prisonNumber/work-and-skills/in-prison-courses-and-qualifications` which has different security requirements, specifically for those users who are not PLP users.

By putting the `inPrisonCoursesAndQualifications` first and (more importantly) the `prisonerList` (root) route last we fix this problem 👍